src/back/controllers/api/routes/categories/categories.spec.js

```diff
@@ -26,18 +34,58 @@ describe('categories handler', () => {
 
         sut.getAll({}, res);
 
-        const onFindCallback = model.find.lastCall.args[1];
+        const onFindCallback = categoryModel.find.lastCall.args[1];
 
         onFindCallback({}, data);
 
         res.json.should.have.been.calledWith(data);
     });
 
-    it('should response with 200 status', () => {
-        const res = { sendStatus: env.stub() };
+    describe('save', () => {
+        let categories;
+        let categoryIds;
+        let req;
+        let res;
+
+        beforeEach(() => {
+            categories = [
+                { _id: 1 },
+                { _id: 2 },
+                { _id: 3 }
+            ];
+            categoryIds = [1, 2, 3];
+            req = {
+                user: { _id: 1 },
+                body: categories
+            };
+            res = {
+                sendStatus: env.stub(),
+                status: env.stub()
+            };
+
+            sut.save(req, res);
+        });
+
+        it('should save user categories', () => {
+            userModel.find.should.have.been.calledWith();
+            updateStub.should.have.been.calledWith({ categories: categoryIds }, sinon.match.func);
```

Comment:
"should save user categories", and you check for just any Function  to haveBeenCalled, could it be more specific?